Add a funding section to README.md (#27)
ngi-nix/summer-of-nix: README.md

[Open an issue](https://github.com/ngi-nix/summer-of-nix/issues) to ask a question in public.
Send us direct messages on Matrix for questions that require privacy.

> ## Funding

Reviewer: ```suggestion
## Acknowledgements
```

> Summer of Nix is supported by [NLnet](https://nlnet.nl/foundation/), with financial backing from the European Commission's [Next Generation Internet](https://ngi.eu/about/) program.

Reviewer: ```suggestion
Summer of Nix is funded by the European Commission's [Next Generation Internet (NGI)](https://www.ngi.eu/) initiative through the [NLNet Foundation](https://nlnet.nl/) and the [NixOS Foundation](https://github.com/NixOS/foundation).
```

Let's just use the same text as in the first paragraph

> [<img src="https://nlnet.nl/logo/banner.svg" alt="NLnet foundation logo" width="30%" />](https://nlnet.nl/foundation/)
> <span style="margin: 10px"></span>
> [<img src="https://nlnet.nl/image/logos/EC.svg" alt="European Commission logo" width="20%" />](https://ngi.eu/about/)

Reviewer: ```suggestion
[<img src="https://nlnet.nl/image/logos/EC.svg" alt="European Commission logo" />](https://ngi.eu/about/)
[<img src="https://nlnet.nl/logo/banner.svg" alt="NLnet foundation logo" />](https://nlnet.nl/foundation/)
```

None of this registers in GitHub markdown, and we're not rendering it anywhere else. Also let's put the EC first.
